### PR TITLE
Tweaks to allow huge flow starts to work under low mem conditions

### PR DESCRIPTION
**Pull request by nicpottier**
Took a while to figure out a way to test this, but printing the highest memory point did the trick.

This should bring the memory needed to start gigantic flow starts way down.

### CODE REVIEW

File: temba/flows/tests.py [hunk 1]
```diff
@@ -3580,11 +3580,13 @@ def test_translations_rule_first(self):
...
         enrolled_group.update_contacts([self.contact], True)
 
-        runs_started = flow.start_msg_flow([self.contact])
+        runs_started = flow.start_msg_flow(Contact.objects.filter(id=self.contact.id))
         self.assertEquals(1, len(runs_started))
         self.assertEquals(2, self.contact.msgs.all().count())
         self.assertEquals('You are in the enrolled group.', self.contact.msgs.all().order_by('-pk')[0].text)
 
+    test_translations_rule_first.active = True
+
```
Comment (ericnewcomer) [inline temba/flows/tests.py]: don't think you want this

**Comment by ericnewcomer** [comment]
All looks kosher.. man i wish there was a sane way to validate memory usage so we dont break this

File: temba/flows/tests.py (hunk 1 above)
Comment (nicpottier) [inline temba/flows/tests.py]: Dah, always forget those.

**Comment by nicpottier** [comment]
Ya, might be the type of thing that belongs in the performance suite that we run now and then.

**Pull request merged**